release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -533,9 +533,31 @@ In the following tables, features are marked with the following statuses:
 
 |====
 
+[discrete]
+=== Workloads deprecated and removed features
+
+.Workloads deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.12 |4.13 |4.14
+
+|`DeploymentConfig` objects
+|General Availability
+|General Availability
+|Deprecated
+
+|====
+
 [id="ocp-4-14-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-14-deployment-config-deprecated"]
+=== DeploymentConfig resources are now deprecated
+
+As of {product-title} 4.14, `DeploymentConfig` objects are deprecated. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
```

Comment:
It is important to avoid mentioning removal of DeploymentConfig from future releases. So something like `however, only security and critical fixes are accepted.` or similar.